Content.Server/Shuttles/Commands/FTLDiskBurnerCommand.cs

```diff
@@ -0,0 +1,176 @@
+using Content.Server.Administration;
+using Content.Server.Labels;
+using Content.Shared.Administration;
+using Content.Shared.Shuttles.Components;
+using Robust.Shared.Console;
+using Robust.Shared.Map.Components;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Utility;
+
+
+
+namespace Content.Server.Shuttles.Commands;
+
+/// <summary>
+/// Creates FTL disks, to maps, grids, or entities.
+/// </summary>
+[AdminCommand(AdminFlags.Fun)]
+
+public sealed class FTLDiskBurnerCommand : LocalizedCommands
+{
+    [Dependency] private readonly IEntityManager _entManager = default!;
+    [Dependency] private readonly IEntitySystemManager _entSystemManager = default!;
+
+    public override string Command => "FTLdiskburner";
```

Comment:
Don't include capital letters in commands, and make the command more plain and descriptive, i.e. something like `ftlcreatedisk`.